[improve][doc] SEO for Transactions (#683)
apache/pulsar-site: docs/txn-how.md

id: txn-how
title: How transactions work?
sidebar_label: "Working principles"

> description: Get a comprehensive understanding of how transactions work in Pulsar.

Reviewer: ```suggestion
description: Learn the working principles of transactions in Pulsar.
```
